apps/dashboard/src/components/activity/components/pagination.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChevronLeft, ChevronRight } from 'lucide-react';
+import { Button } from '@/components/primitives/button';
+
+interface PaginationProps {
+  offset: number;
+  limit: number;
+  hasMore: boolean;
+  onOffsetChange: (newOffset: number) => void;
+}
+
+export function Pagination({ offset, limit, hasMore, onOffsetChange }: PaginationProps) {
```

Comment:
I strongly suggest reusing the pagination component from the workflow table, even if the Activity API doesn't allow it yet. We will isolate the API's compatibility layer on the hook level and then refactor it later.

Reply:
I initially tried, but the difference is too big and makes a lot of weird props and if statements. I would prefer to keep this one for the arrow navigations. We might need this also on the subscribers page.